build.cake

```diff
@@ -133,7 +136,7 @@ Task("UnitTest")
         try 
         {
             DotNetCoreTool(project,
-                "xunit",  "-fxversion 2.0.0 --no-build -parallel none -configuration " + 
+                "xunit",  "-fxversion 2.1.0 --no-build -parallel none -configuration " + 
```

Comment:
dotnet xunit is deprecated btw. Use dotnet test.

Reply:
OK, will try that, thanks for letting us know. I think the reason why we were using dotnet xunit, because dotnet test didn't support adding configuration and other parameters.

Reply:
you should be able to add parameters to dotnet test...?

`dotnet test .\foo.csproj -c Release /p:SomeOtherParam=foo`